pkg/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "pocketsizefund"
+version = "0.1.1"
```

Comment:
I assume SemVer is required? If so that's fine but I'm broadly in favor of simple continuous incrementing rather than differentiating between major/minor/patch.